### PR TITLE
Help with the prerequisites

### PR DESCRIPTION
**Pull request by AxelHamburch**
An help page for the prerequisites, outsourced to a separate page.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,6 +54,8 @@ Balance Notification            |  Transaction Summary
 3. **LNbits Wallet:** Access your LNbits API key (read-only).
 4. **Telegram Bot:** Create a Telegram bot via [BotFather](https://t.me/BotFather) and obtain your bot token.
 5. **Chat ID:** Use the [@userinfobot](https://t.me/userinfobot) on Telegram to find your User ID = chat ID.
+
+Hilfe für die letzten drei Voraussetzungen findet iher hier:
```
Comment (DoktorShift) [inline README.md]: please change to englisch

File: README.md [hunk 2]
```diff
@@ -165,6 +167,7 @@ You can check whether the forwarding works, for example, with this: https://dnsc
 The SSL certificate e.g. with this: https://www.sslshopper.com/ssl-checker.html
 
 ---
+
 ### Step 5: Telegram Bot Webhook Setup
 To enable inline commands (like `/balance`, `/transactions`, `/info`), connect your Telegram bot to the app:
```
Comment (DoktorShift) [inline README.md]: inline command /help is missing

File: prerequisites help.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+If you host your own LNbits server publicly, you also have your own domain. You can set up additional subdomains with domain hosting providers. If you don't have your own domain and use someone else's LNbits server, you can use [duckdns.org](https://www.duckdns.org/). There you can get a free subdomain in the format `mysubdomain.duckdns.org`. All you have to do is point the domain to the IP address of the VPS on which Naughtify is installed and set up Caddy accordingly. That's it. 
+
+### 3. LNbits - The Wallet invoice key and lighting address
+
+LNbits is a wallet and account system. Each account can have multiple wallets and add extensions for different functions. Below is a brief explanation of how to create a wallet, receive the wallet invoice key and generate a static LNURLpay link with a Lightning⚡ address.
+
+3.1 Create an account with an LNbits server of your choice
+
+E.g. at the official demo server [demo.lnbits.com](https://demo.lnbits.com) or at LNbits.de [LNbits.de](https://LNbits.de).
```
Comment (DoktorShift) [inline prerequisites help.md]: Please change to this:

E.g. at the official demo server [demo.lnbits.com](https://demo.lnbits.com) or at community driven TimeCatcher Server [timecatcher.lnbits.de](https://timectcher.lnbits.de).

File: README.md (hunk 1 above)
Comment (AxelHamburch) [reply README.md]: was outdated

File: README.md (hunk 2 above)
Comment (AxelHamburch) [reply README.md]: done

File: prerequisites help.md (hunk 3 above)
Comment (AxelHamburch) [reply prerequisites help.md]: done

**Comment by AxelHamburch** [comment]
done

**Pull request merged**